doc/api/rcsetup_api.rst

```diff
@@ -0,0 +1,12 @@
+*****
```

Comment:
The length of the `****` needs to match that of `rcsetup`; that's causing a warning from sphinx.